base/applications/control/control.c

```diff
@@ -34,6 +39,35 @@ VOID
 WINAPI
 Control_RunDLLW(HWND hWnd, HINSTANCE hInst, LPCWSTR cmd, DWORD nCmdShow);
 
+static BOOL
+IsSwitch(LPCWSTR Switch, LPCWSTR Arg)
+{
+    if (*Arg == '/' || *Arg == '-')
+    {
+        return !lstrcmpiW(Arg+1, Switch);
+    }
+    return FALSE;
```

Comment:
```suggestion
    return ((*Arg == '/' || *Arg == '-') && !lstrcmpiW(Arg+1, Switch));
```

Reply:
This is personal preference, and I see nothing wrong with the original code?

Reply:
Sometimes I get yelled at for too much on one line, then when I break it up that is wrong too 🤣

Reply:
do as you wish indeed.